consoles/localXvnc.pm

```diff
@@ -38,6 +38,7 @@ sub callxterm ($self, $command, $window_name) {
     die('Missing "Xvnc"') unless which('Xvnc');
     die('Missing "icewm"') unless which('icewm');
     die('Missing "xterm"') unless which('xterm');
+    $command = $self->{args}->{log} ? "script -f hardware-console-log.txt -c \"$command\"" : $command;
```

Comment:
This would be a bit easier to read in my opinion:
```suggestion
    $command = "script -f hardware-console-log.txt -c \"$command\"" if $self->{args}->{log};
```

Reply:
Implemented.

Reply:
> `script` is part of `util-linux` so we likely don't have to add it as dependency.

@Martchus FYI `script` has been used long time ago in `backend/svirt.pm:272

https://github.com/os-autoinst/os-autoinst/blob/ae94c4bd4c48eb9b8cc0e09b1728c886d0f27c95/backend/svirt.pm#L267-L279